app/models/acl_dao.rb

```diff
@@ -0,0 +1,72 @@
+class AclDao
+  attr_accessor :access_token, :client_id, :base_url
+
+  def initialize(token, client_id, base_url)
+    @access_token = token
+    @client_id = client_id
+    @base_url = base_url
+  end
+
+  def get_role(user_id)
+
+    # Sets whether the user is a specific role type
+    # Note: user could be all three.
+
+    results = search_acls_by_user(user_id, 1)
+    noPages = (results['hits']/2000).ceil
+    items = Array.wrap(results['items'])
+
+    (2..noPages+1).each { |i|
+      results = search_acls_by_user(user_id, i)
+      items += Array.wrap(results['items'])
+    }
+
+
+    roles = Set.new
+    items.each do |item|
+      roles << "daac_curator" if item['name'].end_with?"DASHBOARD_DAAC_CURATOR"
+      roles << "arc_curator" if item['name'].end_with?"DASHBOARD_ARC_CURATOR"
+      roles << "admin" if item['name'].end_with?"DASHBOARD_ADMIN"
+    end
+
+    #
+    # returns the highest level role
+    #
+    return "admin" if (roles.include? "admin")
```

Comment:
the parens around the roles check should be removed https://github.com/rubocop-hq/ruby-style-guide#no-parens-around-condition